app/shared_features/edit_set/EditTextModal.js

```diff
@@ -164,8 +165,17 @@ class EditTextModal extends Component {
 
     // TODO: grab the blue color for cancel from a global stylesheet
     _renderNavigation() {
-        if (Platform.OS === 'ios') {
+        if (Platform.OS === 'ios' && !Device.isiPhoneX()) {
```

Comment:
- [ ] make it if device is iphonex else if platform == ios to simplify logic so you don't do that extra check